be/src/common/config.h

```diff
@@ -283,10 +283,7 @@ DECLARE_Bool(doris_enable_scanner_thread_pool_per_disk);
 DECLARE_mInt64(doris_blocking_priority_queue_wait_timeout_ms);
 // number of scanner thread pool size for olap table
 // and the min thread num of remote scanner thread pool
-DECLARE_Int32(doris_scanner_thread_pool_thread_num);
-// max number of remote scanner thread pool size
-// if equal to -1, value is std::max(512, CpuInfo::num_cores() * 10)
-DECLARE_Int32(doris_max_remote_scanner_thread_pool_thread_num);
+DECLARE_mInt32(doris_scanner_thread_pool_thread_num);
```

Comment:
I suggest not removing the old config, for compatibility.
Because there may some user who set these configs already, we have to make sure it can keep same value after upgrade